docs.yaml

```diff
@@ -0,0 +1,35 @@
+title:    Java Driver for Apache Cassandra
+summary:  High performance Java client for Apache Cassandra
+homepage: http://datastax.github.io/java-driver/
+sections:
+  - title:     Features
+    prefix:    /features
+    sources:
+      - type:  markdown
+        files: 'features/**/*.md'
+  - title:     API docs
+    prefix:    /api
+    type:      javadoc
+    directory:
+      - 'driver-core/src/main'
+      - 'driver-dse/src/main'
+      - 'driver-mapping/src/main'
+    namespace: Cassandra
+links:
+  - title: Code
+    href:  https://github.com/datastax/java-driver/
+  - title: Docs
+    href:  http://datastax.github.io/java-driver/
+  - title: Issues
+    href:  https://datastax-oss.atlassian.net/browse/JAVA/
+  - title: Mailing List
+    href:  https://groups.google.com/a/lists.datastax.com/forum/#!forum/java-driver-user
+  - title: IRC Channel
+    href:  irc://irc.freenode.net/datastax-drivers
+  - title: Releases
+    href:  https://github.com/datastax/java-driver/releases
+versions:
+  - name: 2.0.10
```

Comment:
You could use HEAD as the red until the release and change to the tag name afterwards